docs/csharp/programming-guide/classes-and-structs/methods.md

```diff
@@ -1,15 +1,17 @@
 ---
 title: "Methods - C# Programming Guide"
 description: A method in C# is a code block that contains a series of statements. A program runs the statements by calling the method and specifying arguments.
-ms.date: 07/20/2015
+ms.date: 03/08/2021
 helpviewer_keywords:
   - "methods [C#]"
   - "C# language, methods"
 ms.assetid: cc738f07-e8cd-4683-9585-9f40c0667c37
 ---
 # Methods (C# Programming Guide)
 
-A method is a code block that contains a series of statements. A program causes the statements to be executed by calling the method and specifying any required method arguments. In C#, every executed instruction is performed in the context of a method. The `Main` method is the entry point for every C# application and it's called by the common language runtime (CLR) when the program is started.
+A method is a code block that contains a series of statements. A program causes the statements to be executed by calling the method and specifying any required method arguments. In C#, every executed instruction is performed in the context of a method.
+
+The `Main` method is the entry point for every C# application and it's called by the common language runtime (CLR) when the program is started. In an application that uses [top-level statements](../main-and-command-args/top-level-statements.md), the `Main` method is implicit.
```

Comment:
I'm fine with how you wrote this. Consider this wording. It feels more correct, but it's also quite a bit more wordy and may be a distraction in this article.

```suggestion
The `Main` method is the entry point for every C# application and it's called by the common language runtime (CLR) when the program is started. In an application that uses [top-level statements](../main-and-command-args/top-level-statements.md), the `Main` method is generated by the compiler and contains all top-level statements.
```